markdown/beginners_guide.md

```diff
@@ -45,28 +61,39 @@ In order to run a Nextflow pipeline from nf-core on your local computer you need
 
 
 1. Install Nextflow
-
-    `curl -s https://get.nextflow.io | bash`
+    ```bash
```

Comment:
Adding a code name after the three backticks is not standard markdown, but works on GitHub, the website and most other places. It enables syntax highlighting for that code block.